src/libraries/System.Private.CoreLib/src/System/Numerics/IBinaryNumber.cs

```diff
@@ -11,7 +11,7 @@ public interface IBinaryNumber<TSelf>
         where TSelf : IBinaryNumber<TSelf>
     {
         /// <summary>Gets an instance of the binary type in which all bits are set.</summary>
-        static abstract TSelf AllBitsSet { get; } // TODO: add the DIM once https://github.com/dotnet/linker/issues/2865 is fixed
+        static virtual TSelf AllBitsSet => ~TSelf.Zero;
```

Comment:
Can we add tests for this?  Hopefully we have tests for all other DIMs, but if not, we should add some for those, too.

Reply:
Sure, I'll look into this.